### DISCUSSION TITLE
Bug Tracker

### DISCUSSION THREAD

**Discussion by deadlyBuzz** [gitextensions/gitextensions#11670] [Ideas]
Would be great if there was an integration with an existing bug or task tracker, or if there was a built in tool like task coach that could be used to track the open items on my repo.

**Comment by RussKie** [comment]
It is not something on our roadmap. You can try to build a plugin to achieve the desired.

**Reply by gerhardol** [reply]
There is for GitHub and a very limited plugin for BitBucket server
